src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -55,16 +57,41 @@ export const INTERFACES = {
   spans: SpansInterface,
 };
 
-class EventEntries extends React.Component {
+const defaultProps = {
+  isShare: false,
+  showExampleCommit: false,
+  showTagSummary: true,
+};
+
+// Custom shape because shared view doesn't get id.
+type EventEntriesOrganization = {
+  id?: string;
+  slug: string;
+  features?: Array<string>;
+};
```

Comment:
```suggestion
type EventEntriesOrganization = {
  slug: string;
  id?: string;
  features?: Array<string>;
};
```

Reply:
we usually define optional props in the end of the object 😉

Reply:
done, thx